site/content/docs/user/configuration.md

```diff
@@ -181,6 +181,8 @@ networking:
   podSubnet: "10.244.0.0/16"
 {{< /codeFromInline >}}
 
+By default, kind uses ```10.244.0.0/16``` pod subnet for IPv4 and ```fd00:10:244::/56``` pod subnet for IPv6.
```

Comment:
```suggestion
By default, `kind` uses ```10.244.0.0/16``` pod subnet for IPv4 and ```fd00:10:244::/56``` pod subnet for IPv6.
```

@BenTheElder single ticks?

Reply:
I had thought of highlighting `kind` as well, but I saw kind being used without highlights in other places. So left it as it is.

Reply:
Ben is the best person to decide, but I think ticks is better

Reply:
kind or KIND but not KinD
`kind` or `KIND` but not KinD

:-)

thanks for documenting this, been a hectic period with https://kubernetes.io/blog/2023/03/10/image-registry-redirect/, the docker hub announcements, the runc cves ...

Reply:
@aojea we could add a note to the contributing guide under docs development